containers/registry-spec-renderer/index.js

```diff
@@ -1,85 +1,136 @@
 const express = require('express')
 const fs = require("fs");
 const handlebars = require("handlebars");
-const { RegistryClient } = require("@giteshk-org/apigeeregistry");
+const {RegistryClient} = require("@giteshk-org/apigeeregistry");
```

Comment:
Is this a reference to a node package for the GAPIC? (I'm unfamiliar with how this gets looked up and were the package is hosted)

Reply:
Yes that is correct. 

At the moment I am hosting it on the Github NPM registry. https://github.com/apigee/registry-experimental/blob/main/containers/registry-spec-renderer/.npmrc

In future once we have this build out by the Google team. We will change this reference.